src/ingest_validation_tools/upload.py

```diff
@@ -370,14 +367,11 @@ def _validate(
     def _get_reference_errors(self) -> dict:
         errors: Dict[str, Any] = {}
         no_ref_errors = self.__get_no_ref_errors()
-        try:
-            multi_ref_errors = self.__get_multi_ref_errors()
-            if no_ref_errors:
-                errors["No References"] = no_ref_errors
-            if multi_ref_errors:
-                errors["Multiple References"] = multi_ref_errors
-        except UnicodeDecodeError as e:
-            errors["Decode Error"] = get_context_of_decode_error(e)
+        multi_ref_errors = self.__get_multi_ref_errors()
```

Comment:
Why did you take this out of the try catch block?

Reply:
Just double checked and yep that's why! Storing the rows on the SchemaVersion helps to avoid reading the TSV rows over and over.